app/ts/library/bundleUtils.ts

```diff
@@ -17,20 +17,43 @@ export const createProvider = async (provider: Signal<ProviderStore | undefined>
 	return flashbotsProvider
 }
 
+
+async function getSimulatedCountsOnNetwork(provider: providers.Web3Provider): Promise<{ [address: string]: bigint }> {
+	try {
+		const { payload } = await provider.send(
+			'interceptor_getSimulationStack',
+			['1.0.0']
+		)
+		const result = payload.reduce((acc: { [address: string]: bigint }, curr: { from: string }) => {
+			curr.from = utils.getAddress(curr.from)
+			if (curr.from in acc) acc[curr.from] += 1n
+			else acc[curr.from] = 1n
+			return acc
+		}, {})
+		return result
+	} catch {
+		return {}
```

Comment:
we shouldn't silently hide all errors. Could console.error it and return {}?

Reply:
Good point, we expect it to fail if the user isn't using Interceptor, but logging it doesn't hurt.